### ISSUE TITLE
Turn off Capture Canvas

### ISSUE THREAD

**Issue by Havoclivekiller** [flamewave000/dragonflagon-arch#48]
**Is your feature request related to a problem? Please describe.**
Whenever I click Enter, it goes to Capture Canvas instead of closing the window. Counter-intuitive. 

**Describe the solution you'd like**
Ability to turn off Capture Canvas completely, not only for the players.

**(Optional) Describe alternatives you've considered**
Or not have it catch the Enter button.

**Comment by flamewave000** [comment]
Is this for the colour picker? Canvas Capture is for capturing the game board as an image

**Comment by Havoclivekiller** [comment]
Yeah, I mean the color picker that captures the color from the canvas.

**Comment by flamewave000** [comment]
@Havoclivekiller I'm having difficulty reproducing this issue. Is there a particular case where it happens to you every time?

**Comment by Havoclivekiller** [comment]
I don't seem to be able to? Maybe FVTT update fixed it? I'll inform you if I catch it.
